runtime/kusama/src/lib.rs

```diff
@@ -1487,7 +1487,7 @@ construct_runtime! {
 		ParasInherent: parachains_paras_inherent::{Pallet, Call, Storage, Inherent} = 54,
 		ParasScheduler: parachains_scheduler::{Pallet, Call, Storage} = 55,
 		Paras: parachains_paras::{Pallet, Call, Storage, Event, Config} = 56,
-		ParasInitializer: parachains_initializer::{Pallet, Call, Storage} = 57,
+		Initializer: parachains_initializer::{Pallet, Call, Storage} = 57,
```

Comment:
@thiolliere could you clarify whether this needs a runtime migration? and if so if it can be avoided? and if not whether it should be included in the PR?

Reply:
the top comment says:

> So users of the Initializer pallet must be careful about the name they used in construct_runtime!. Hence the runtime-migration label, which might not be needed depending on the configuration of the Initializer pallet.
> 
> Kusama and westend both use ParasInitializer in construct_runtime!, thus must be migrated

We would need migration if kept naming the pallet `ParasInitialize` in construct_runtime, but the author finally renamed those pallet to `Initializer` in https://github.com/paritytech/polkadot/pull/3416/commits/ece08ac315452a105e3f17e657a8f6c9c782824d
(cc @ferrell-code could you update the top comment to reflect the final version ?)

So for Kusama and Westend the name of the pallet is now matching the previously hard-coded prefix in decl_storage call: `Initializer`.
And polkadot doesn't use initializer pallet.

So no need for any migration.